integration-tests/tests/test_wallet_rpc.py

```diff
@@ -39,14 +39,45 @@ def test_tx_(self, method, params):
 
     def test_create_multi_transaction(self):
         response = self.wallet_create_multi_transaction()
+        self.verify_is_valid_json_rpc_response(response)
         
         # how to create schema:
         # from schema_builder import CustomSchemaBuilder
         # CustomSchemaBuilder(method).create_schema(response.json())
         
-        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction", "r") as schema:
+        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction/transferTx_positive", "r") as schema:
             jsonschema.validate(instance=response.json(), schema=json.load(schema))
 
+    @pytest.mark.parametrize(
+        "method, changed_values, expected_error_code, expected_error_text",
+        [
+            (
+                    "transferTx_value_not_enough_balance",
+                    {'value': '0x21e438ea8139cd35004'}, -32000, "Insufficient funds for gas",
+            ),
+            (
+                    "transferTx_from_from_invalid_string",
+                    {'from': 'some_invalid_address'}, -32602, "cannot unmarshal hex string without 0x prefix",
+            ),
+        ],
+    )
+    def test_create_multi_transaction_validation(self, method,
+                                                 changed_values,
+                                                 expected_error_code, expected_error_text):
+        response = self.wallet_create_multi_transaction(**changed_values)
+        self.verify_is_json_rpc_error(response)
+        actual_error_code, actual_error_text = response.json()['error']['code'], response.json()['error']['message']
+        if expected_error_code != actual_error_code:
+            raise AssertionError(
+                f"got code: {actual_error_code} instead of expected: {expected_error_code}"
+        )
+        if expected_error_text not in actual_error_text:
+            raise AssertionError(
+                f"no required text in message: {actual_error_text} instead of expected: {expected_error_text}"
+            )
+        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction/transferTx_error", "r") as schema:
```

Comment:
sorry, didn't notice on first review, we can use https://github.com/status-im/status-go/blob/develop/integration-tests/tests/test_cases.py#L45 here, I will also update other tests

Reply:
can you elaborate please?

Reply:
I meant to use `verify_json_schema` method